networkmanager/networkmanager.go

```diff
@@ -336,22 +336,6 @@ func (manager *NetworkManager) RemoveInstanceFromNetwork(instanceID, networkID s
 	return manager.deleteInstanceNetworkFromCache(instanceID, networkID)
 }
 
-// GetInstanceIP return instance IP address.
-func (manager *NetworkManager) GetInstanceIP(instanceID, networkID string) (ip string, err error) {
```

Comment:
Let's keep this method, could be useful in the future.

Reply:
Done